public/js/components/AtomEdit/CustomEditors/StoryQuestionFields/Question.js

```diff
@@ -25,12 +28,19 @@ export class StoryQuestionsQuestion extends React.Component {
   }
 
   render () {
+    const answersCount = (this.props.fieldValue && this.props.fieldValue.answers) ? this.props.fieldValue.answers.length : 0;
```

Comment:
Lodash is included, so you can use [`_.get`](https://lodash.com/docs/4.17.4#get) to make this cleaner.

Reply:
💯 amazing